translation/japanese/patterns/trusted-committer.md

```diff
@@ -1,10 +1,6 @@
-{% hint style="info" %}
-このパターンで紹介されている "**Trusted Committer**" という語は、"**信頼されたコミッター**" や "**トラステッドコミッター**" と訳されます。本パターンブックでの用語の使い分けに関しては「[トラステッドコミッターというロールと呼称に関して](#trusted_committer)」をご覧ください。
-{% endhint %}
-
 ## Title
 
-信頼されたコミッター (トラステッドコミッター)
+トラステッドコミッター
```

Comment:
@yuhattor This looks like you had multiple translations for TC here?

The main one, and then a second one in brackets?

Note that you can use the '## Alias' section for such alternative titles.

Reply:
TC is a very important word, and we had a deep discussion in the community on how to express it.
Long story short, we decided not to translate "Trusted Committer" as suggested by ICS. Japanese has katakana, a character that represents a sound.

So we are okay with using one word here, but if we need to alias it in the future for other terms, we will do so

Reply:
One day you need to tell me more about what "katakana" means. Sounds great for sure :)

But for now, let's resolve this thread.